src/protocol/thread/pause.ts

```diff
@@ -484,6 +484,7 @@ export class Pause {
       rv = await this.sendMessage(client.DOM.repaintGraphics, {});
     } catch (e) {
       console.error("DOM.repaintGraphics failed", e);
+      throw e;
```

Comment:
This change isn't necessary, as the caller can test for null.  This change is also incorrect, because the repaintGraphicsWaiter won't get resolved and later calls to repaint the graphics will get stuck waiting forever.